sql/firefox_accounts_derived/fxa_users_services_last_seen_v1/metadata.yaml

```
@@ -0,0 +1,11 @@
+friendly_name: FxA Users Services Last Seen
+description: Usage aggregations per FxA user per FxA service per day over a 28-day window
+owners:
+  - jklukas@mozilla.com
+labels:
+  application: fxa
+  incremental: true
+  schedule: daily
+scheduling:
+  dag_name: bqetl_fxa_events
+  depends_on_past: true
```

```suggestion
  depends_on_past: true
  start_date: '2019-10-08'
```

https://github.com/mozilla/telemetry-airflow/blob/caa17a72b905adc8e975d9310814032855abea87/dags/fxa_events.py#L170